go.mod

```diff
@@ -190,3 +190,5 @@ replace (
 
 	k8s.io/kubectl => github.com/openshift/kubernetes-kubectl v0.0.0-20220124124213-df787b5b471c
 )
+
+replace github.com/code-ready/machine v0.0.0-20220727160217-7bf0dd8c2d7b => github.com/anjannath/machine v0.7.1-0.20220829061903-fb47540d2400
```

Comment:
TODO: this needs to be changed

Reply:
will drop this commit and vendor updated `code-ready/machine` once the PR https://github.com/code-ready/machine/pull/66 is merged

Reply:
this is taken care of now, PR on machine side is merged and this PR upgrades the code-ready/machine module